packages/xsnap/src/build.js

```diff
@@ -238,7 +238,7 @@ const buildXsnap = async (platform, force, { fs, make }) => {
 
   const configEnvs = [
     `XSNAP_VERSION=${pkg.version}`,
-    `CC=cc "-D__has_builtin(x)=1"`,
```

Comment:
This causes warnings on Mac. Might be necessary on Linux. Regardless, best to settle the argument in the respective platform Makefile if it crops up.